apps/emqx_bridge/src/emqx_bridge_api.erl

```diff
@@ -461,9 +463,9 @@ schema("/bridges_probe") ->
         }
     }.
 
-'/bridges'(post, #{body := #{<<"type">> := BridgeType0, <<"name">> := BridgeName} = Conf0}) ->
-    BridgeType = upgrade_type(BridgeType0),
-    case emqx_bridge:lookup(BridgeType, BridgeName) of
+'/bridges'(post, #{body := #{<<"type">> := BridgeType, <<"name">> := BridgeName} = Conf0}) ->
+    ActionType = upgrade_type(BridgeType),
+    case emqx_bridge:lookup(ActionType, BridgeName) of
```

Comment:
Just a note for posterity: we should change `lookup` to operate only on v1 types.